src/lib/y2storage/encryption_method.rb

```diff
@@ -76,16 +90,28 @@ def to_human_string
       _(@label)
     end
 
+    # Compare two encryption methods
+    #
+    # @param other [Y2Storage::EncryptionMethod]
+    # @return [Boolean] true if compared encryption methods have the same class and id; false if not
     def ==(other)
       other.class == self.class && other.id == id
     end
 
     alias_method :eql?, :==
 
+    # Whether the given value matches with the method id
```

Comment:
NP: I would say "Whether the given value matches the symbol representation (id) of the encryption method"